lib/operations/misc.js

```diff
@@ -101,22 +142,91 @@ misc.difference = function (x, y) {
 }
 
 /**
- * Computes the distance of the bounds of an interval
+ * Computes the width of the interval. Also known as diameter. 
  * @param {Interval} x
  * @returns {number}
  */
-misc.width = function (x) {
+misc.wid = function (x) {
```

Comment:
Is there a reason to change `width` to `wid`? I'd rather make this an alias to `width` if you want to use 3 letters for the name, I can do the same for `mul,multiply`, `sub,subtract` and so on